dust_emissivity/fit_sed.py

```diff
@@ -298,7 +298,8 @@ def f(p):
                    for n,x in zip(('T','beta','N'),guesses)
                    ]
         guesspars = lmfit.Parameters()
-        guesspars.update(OrderedDict(parlist),)
+        for par in parlist:
+            guesspars.add_many(par)
```

Comment:
should this just be `add`, since we're looping over the parameters?

Reply:
The Parameters function `add` does not accept Parameter objects that have already been defined. It would be simpler to not define the parlist separately and just add the Parameter objects to guesspars as they are defined.